__tests__/app.test.js

```diff
@@ -263,4 +263,30 @@ describe('Delete request /api/comments/:comment_id', ()=>{
       expect(response.body.msg).toBe('Bad request');
     })
   })
+})
+
+describe('/api/users', () => {
+  test('GET:200 sends an array of all users to the client', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.length).toBe(4);
+        response.body.forEach((user) => {
+          expect(user).toMatchObject({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+          });
+        });
+      });
+  })
+  test('GET:404 error when given an invalid URL',() => {
+    return request(app)
+    .get('/api/Fiji')
```

Comment:
You tested this already on line 29 (ish) with your `api/combatwombat` (excellent endpoint name however)